code/drasil-example/hghc/lib/Drasil/HGHC/HeatTransfer.hs

```diff
@@ -68,9 +68,9 @@ htTransCladFuelEq = (exactDbl 2 `mulRe` sy cladCond `mulRe` sy gapFilmCond) $/ (
 hghc :: CommonConcept
 hghc = dcc' "hghc" (cn "HGHC") "HGHC program" "HGHC"
 
-nuclearPhys, fp :: NamedChunk
-nuclearPhys = nc "nuclearPhys" (nounPhraseSP "nuclear physics")
-fp = nc "fp" (cn "FP")
+nuclearPhys, fp :: IdeaDict
```

Comment:
Note, for example, that *these* type changes are fine, because it really is the new type. (The `Nothing`s are still a problem).